src/daipecore/container/container_refresher.py

```diff
@@ -0,0 +1,19 @@
+from daipecore.container.ConfigsWatcherThread import ConfigsWatcherThread
+from daipecore.container.WatcherLogger import WatcherLogger
+
+
+def watch_configs(configs_dir: str, init_container: callable):
+    watcher_logger = WatcherLogger()
+
+    def prepare_container():
+        try:
+            init_container()
+            watcher_logger.info("New container ready")
+        except Exception as e:
+            watcher_logger.error("Container initialization failed")
+            watcher_logger.debug(str(e))
+
+    watcher_thread = ConfigsWatcherThread(configs_dir, watcher_logger, prepare_container)
+    watcher_thread.start()
```

Comment:
Is there a chance that we accidentaly spawn multiple threads instead of just one?